.github/workflows/trackdeploys.yaml

```diff
@@ -0,0 +1,55 @@
+name: Track semVer through environments
+
+on:
+  schedule:
+    - cron: '0 */3 * * *' # run every 3 hours and on push
+  workflow_dispatch: {}
+
+jobs:
+  get_versions:
+    name: Get semVer for environments
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_BUCKET_SA_KEY }}
+          export_default_credentials: true
+      - name: Use gcloud copy endpoints.json
+        run: gsutil cp gs://deployment-tracking/endpoints.json .
+      - shell: bash
+        run: |
+          endPoints+=( "https://jade.datarepo-dev.broadinstitute.org/configuration" \
+            "https://data.alpha.envs-terra.bio/configuration" \
+            "https://data.staging.envs-terra.bio/configuration" \
+            "https://data.terra.bio/configuration")
+
+          for i in ${endPoints[@]}; do
+            if [[ "$i" == *"dev"* ]]
+            then
+             env=dev
+            elif [[ "$i" == *"alpha"* ]]
+            then
+             env=alpha
+            elif [[ "$i" == *"staging"* ]]
+            then
+             env=staging
+            elif [[ "$i" == *"data.terra.bio"* ]]
+            then
+             env=production
+            else
+             echo "No env found"
+            fi
+            date=$(date +%Y-%m-%d\ %H:%M:%S)
+            semVer=$(curl -s $i | jq -r '.semVer')
+            cat <<< $(jq --arg env "$env" \
+              --arg date "$date" \
+              --arg semVer "$semVer" \
+              '.data += [{"env": "'"$env"'", "date": "'"$date"'", "semVer": "'"$semVer"'"}]' endpoints.json) \
+              > endpoints.json
```

Comment:
I am not sure how well defined `cat <<< $(... $file) > $file` is.

Reply:
unsure i stole it from here https://stackoverflow.com/questions/36565295/jq-to-replace-text-directly-on-file-like-sed-i

Reply:
Looks like it's literally the string to the right

Reply:
Oh, you meant like order of operations?

Reply:
just checking: this prints the contents of endpoints.json to the console?  Be good if it did for tracking in the logs

Reply:
it doesn't since its redirected to the file

Reply:
Yes, I meant order of operations. It redirects the output of the jq command that reads in a file and outputs it to that same file. It looks like it works okay though.